### PR TITLE
chore: fix size-limit action

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
大佬, v4也要

File: package.json [hunk 1]
```diff
@@ -316,13 +317,17 @@
   "size-limit": [
     {
       "path": "./dist/antd.min.js",
-      "limit": "375 kB"
+      "limit": "377 kB"
```
Comment (afc163) [inline package.json:320]:
```suggestion
      "limit": "385 kB"
```

File: package.json [hunk 2]
```diff
@@ -316,13 +317,17 @@
...
+    },
+    {
+      "path": "./dist/antd-with-locales.js",
```
Comment (afc163) [inline package.json:323]:
```suggestion
      "path": "./dist/antd-with-locales.min.js",
```

**Comment by afc163** [comment]
surge 挂了， https://github.com/sintaxi/surge/issues/496

File: package.json [hunk 3]
```diff
@@ -316,13 +317,17 @@
   "size-limit": [
     {
       "path": "./dist/antd.min.js",
-      "limit": "375 kB"
+      "limit": "385 kB"
+    },
+    {
+      "path": "./dist/antd-with-locales.min.js",
+      "limit": "510 kB"
```
Comment (afc163) [inline package.json:324]:
```suggestion
      "limit": "440 kB"
```

File: package.json [hunk 4]
```diff
@@ -316,13 +317,17 @@
   "size-limit": [
     {
       "path": "./dist/antd.min.js",
-      "limit": "375 kB"
+      "limit": "385 kB"
```
Comment (afc163) [inline package.json:320]:
```suggestion
      "limit": "390 kB"
```

**Review by zombieJ — APPROVED** [review]

**Pull request merged**